packages/smock/README.md

```diff
@@ -51,40 +51,6 @@ const config: HardhatUserConfig = {
 export default config
 ```
 
-## Table of Contents
```

Comment:
GitHub handles ToC relatively well now so we can just get rid of this.

Reply:
It does? How/where?

Reply:
<img width="338" alt="Screen Shot 2021-05-18 at 10 54 12 AM" src="https://user-images.githubusercontent.com/14298799/118673963-6e30d100-b7c7-11eb-9cf2-9800bcdd2bf4.png">

Reply:
oh nice!